shim/interfaces.go

```diff
@@ -80,6 +80,15 @@ type ChaincodeStubInterface interface {
 	// If the key does not exist in the state database, (nil, nil) is returned.
 	GetState(key string) ([]byte, error)
 
+	// GetMultipleStates retrieves the values of the specified keys from the ledger.
+	// It has similar semantics to the GetState function regarding read-your-own-writes behavior,
+	// meaning that updates made by a previous PutState operation within the same chaincode
+	// session are not visible to this function. As a result, such keys will return nil without raising
+	// an error. If the keys do not exist in the state database, the function will also return (nil, nil).
```

Comment:
This sentence seems slightly confusing. It is not explicit about which keys need to be missing for (nil, nil) to be returned. Maybe it is redundant since the very next sentence explicitly describes the structure of the returned values, and which ones will be nil.

Reply:
fixed